docs/user/dashboard/share-dashboards.asciidoc

```diff
@@ -0,0 +1,27 @@
+[[share-dashboards]]
+== Share dashboards
+
+[[embedding-dashboards]]
+Share your dashboard outside of {kib}.
```

Comment:
Maybe include a screenshot?

Reply:
What should I include in the screenshot?